nvtabular/ops/reduce_dtype_size.py

```diff
@@ -63,14 +64,14 @@ def compute_output_schema(self, input_schema, selector, prev_output_schema=None)
             column = input_schema[column]
 
             dtype = column.dtype
-            if np.issubdtype(column.dtype, np.integer):
+            if column.dtype.elemtype.value == "int":

```

Comment:
This would be good to wrap up in a helper method so that calling code doesn't need to know the details of the dtype implementation to check whether or not a dtype is an integer type